fundbot/plugins/fund/search_impl.py

```diff
@@ -25,8 +25,7 @@ def cache_add(search_args: list, result: str) -> str:
 
 def draw_page(arg_list):
     try:
-        arg_list.pop(len(arg_list) - 1)
-        return int(arg_list[-1]) - 1
+        return int(arg_list.pop(len(arg_list) - 1)) - 1

```

Comment:
does it not remove the last element default?
arg_list.pop() ??

Reply:
still have bug, but I've fixed it.
should pop after `int arg_list[-1]` succeeds.